packages/cardano-services/src/cli.ts

```diff
@@ -114,6 +123,7 @@ withCommonOptions(
   .addOption(
     new Option('--drop-schema <true/false>', ProjectorOptionDescriptions.DropSchema)
       .default(false)
+      .env('DROP_SCHEMA')
```

Comment:
This should be committed as a separately, since it's an isolated fix, given our standards are to include ENVs for each command line option.

Reply:
I'll fix this and the two commits description (chore -> refactoring) after rebasing